src/lib/stringIndices.js

```diff
@@ -0,0 +1,16 @@
+  let stringIndices = (userString, userNumber) => {
+    let newArr = []
+    let separate = ""
+
+    if( userString.indexOf(" ") == -1 ){
+      let newArr = Array.from(userString)
+      newArr.unshift(" ")
+      return newArr[userNumber]
+    }
+    if(userString.indexOf(" ") !== -1){
+      let separate = userString.split(' ')
+      let newArr = Array.from(separate)
+      newArr.unshift(" ")
+      return newArr[userNumber]
+    }
+}
```

Comment:
Right now the first if makes it so when the user only inputs one word as the string, the userNumber = letter of that word. Is this what you intended? To me it would make sense that if they entered one word, and the userNumber was > 1 it should return error or undefined?